src/modules/launcher/README.md

```diff
@@ -1 +1,69 @@
-TO DO UPDATE
+# PowerToys Run
+PowerToys Run is a quick launcher for power users that has additional
+capabilities without sacrificing performance. It is open source and
+modular for additional plugins. Press Alt+Space and start typing!
+
+![](/src/modules/launcher/images/QuickStart.gif)
+
+# 1\. Get Started 
+## 1\.1 Features
+Search for applications, folders, or files
+![](/src/modules/launcher/images/Features.gif)
+
+Search for running processes (previously
+[WindowWalker](https://github.com/betsegaw/windowwalker/))
+
+![](/src/modules/launcher/images/FeaturesWindowWalker.gif)
+
+Clickable buttons with keyboard shortcuts
+
+![](/src/modules/launcher/images/FeaturesButtons.gif)
+
+Invoke Shell Plugin using "\> "
+
+![](/src/modules/launcher/images/FeaturesShell.gif)
+
+Do a simple calculation using calculator
+
+![](/src/modules/launcher/images/FeaturesCalculator.gif)
+
+## 1\.2 Settings 
+  | **Settings**   |               **Action** |
+  | --------------- | ------------------------------------------------------------------------------- |
+  | Maximum number of results  |   Maximum number of results shown without scrolling on PowerToys Run |
+  | Open PowerToys Run shortcut |  The keyboard shortcut to open/hide PowerToys Run |
+
+## 1\.3 Keyboard Shortcuts
+  | **Shortcuts**   |   **Action** |
+  | ------------------ | ---------------------------------------------------------------------------------|
+  | Alt+Space |         Open or hide PowerToys Run |
+  | Esc |               Hide PowerToys Run |
+  | Ctrl+Shift+Enter |  (Only applicable to applications) Open the selected application as administrator |
+  | Ctrl+Shift+E |      (Only applicable to applications and files) Open containing folder in File Explorer |
+  | Ctrl+C |            (Only applicable to folders and files) Copy path location |
+  | Tab |               Navigate through the search result and context menu buttons |
+
+# 2\. Known Issues
```

Comment:
@jyuwono I think you might have missed this: https://github.com/microsoft/PowerToys/issues/2456

Reply:
@somil55 nice catch! I've added it